src/components/ContentPreview.tsx

```diff
@@ -1,29 +1,161 @@
 import React, { FC } from 'react';
 import { useTranslation } from 'react-i18next';
-import { Typography } from '@material-ui/core';
-import { Preview, resourceType } from '../types/content.types';
+import { Button, Typography } from '@material-ui/core';
+import { resourceType, SupportedFileTypes } from '../types/content.types';
 import styled from 'styled-components';
+import { Resource } from '../types/resource.types';
+import { API_PATHS, API_URL } from '../utils/constants';
+import { Alert } from '@material-ui/lab';
+import CloudDownloadIcon from '@material-ui/icons/CloudDownload';
 
-const ImageWrapper = styled.img`
+const StyledImage = styled.img`
   height: 100%;
 `;
 
-const VideoWrapper = styled.video`
+const StyledVideo = styled.video`
   height: 100%;
 `;
 
-const ContentPreview: FC<{ preview: Preview }> = ({ preview }) => {
-  const { t } = useTranslation();
+const InformationAndDownloadWrapper = styled.div`
+  display: block;
+  width: 27rem;
+  text-align: center;
+`;
+
+const StyledAlert = styled(Alert)`
+  margin-bottom: 2rem;
+`;
+
+const documentTypeFromMime = (resource: Resource): string => {
```

Comment:
Flyttes til fil for seg selv?

Reply:
tja. Jeg stemmer for det. Denne filen er jo ikke så stor i seg selv, men man må scrolle endel for å komme til hovedinnholdet.

Reply:
Fikset